components/renderers/widgets/widgeta/widgeta-renderer.ts

```diff
@@ -0,0 +1,57 @@
+///<reference path="../../../references.ts"/>
+
+class WidgetARenderer implements JSONForms.IRenderer {
```

Comment:
Jonas told me to implement one basic widget (widgeta) and other showing controls instead of lines of text (widgetb). I'm not sure if widgeta is what Jonas wanted, so we can not include it by the moment.